dts/arm/nxp/nxp_k6x.dtsi

```diff
@@ -460,7 +460,6 @@
 			interrupts = <83 0>, <84 0>, <85 0>;
 			interrupt-names = "TX", "RX", "ERR_MISC";
 			status = "disabled";
-			local-mac-address = [00 00 00 00 00 00];
```

Comment:
Based on existing practice in Linux I think a zeroed local-mac-address should still be handled in a way that doesn't cause its presence to use that as the MAC address at runtime.  Equivalent to not having the property would make sense, and avoid having out-of tree overlays that specify the null address misbehave at runtime.

Reply:
Add some macros to handle a "null' mac-address case.